Documentation/operations/system_requirements.rst

```diff
@@ -77,6 +77,7 @@ LinuxKit_                  all
 Opensuse_                  Tumbleweed, >=Leap 15.4
 `RedHat Enterprise Linux`_ >= 8.0
 Ubuntu_                    >= 18.04.3
+`Bottlerocket OS`_         all
```

Comment:
Looks like the rest of this table is sorted by distribution name, in alphabetical order. Could you please move your entry?